verification/curator-service/api/src/geocoding/adminsFetcher.ts

```diff
@@ -0,0 +1,95 @@
+import { GeocodeResult, Resolution } from './geocoder';
+
+import axios from 'axios';
+
+/**
+ * Mapbox administrative area fetcher using the boundaries API.
+ * https://www.mapbox.com/boundaries/.
+ *
+ * Some geocoding results do not contain the full administrative area details,
+ * such details can be subsequently fetched using the boundaries API by doing a
+ * tilequery fetch of the center of the geocode result with the administrative
```

Comment:
> of the center of the geocode result

Can it be looked up by name, too? If our geocode result for "Adams, Ohio, US" is just the center of Ohio, that's unlikely to give us the right county.

Reply:
no the tilequery api that boundaries is based on only works with lat lng as input.

Reply:
Then does this address geocoding "Miami-Dade [County], Florida"? Or do we have a :chicken: <-> :egg: situation?

Reply:
For the "no results case" (I suppose Miami-dade is such an example?) I added the ability to just provide a geocode ourselves manually.

Reply:
OK, I think we'll have to consider this more (feel free to submit this PR).

There's lots of US data that's organized by county -- so the data contains admin2 (county), admin1 (state), and country. For bulk upload, IIUC, this basically means the only solution to getting admin2 properly into the data is to somehow manually geocode the raw CSV data and supply the lat/long directly as columns. Or, we could just live with bad lat/long and "force" the admin2 name, as I think you're describing for manual upload.